cmake/protobuf.cmake

```diff
@@ -39,7 +39,9 @@ if(gRPC_PROTOBUF_PROVIDER STREQUAL "module")
     endif()
     if(TARGET protoc)
       set(_gRPC_PROTOBUF_PROTOC protoc)
-      set(_gRPC_PROTOBUF_PROTOC_EXECUTABLE $<TARGET_FILE:protoc>)
+      if(NOT DEFINED _gRPC_PROTOBUF_PROTOC_EXECUTABLE)
+        set(_gRPC_PROTOBUF_PROTOC_EXECUTABLE $<TARGET_FILE:protoc>)
```

Comment:
Btw, this is a different logic than we use for _gRPC_CPP_PLUGIN
https://github.com/grpc/grpc/blob/cd56d6664395f93dcc3ff1564df2d0d034275190/templates/CMakeLists.txt.template#L311
we should probably unify. Can you rely on CMAKE_CROSSCOMPILING here?

Reply:
Good suggestion. I've updated the PR accordingly.

Reply:
👍